### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Set rights datadir

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -89,14 +89,18 @@ ynh_script_progression --message="Configuring PHP-FPM..." --weight=1
...
 #=================================================
+ynh_script_progression --message="Creating a data directory..." --weight=1
 
-content_path=/home/yunohost.app/$app
-mkdir -p $content_path
-ynh_app_setting_set --app=$app --key=content_path --value=$content_path
-chmod -R 750 $content_path
-chown -R $app $content_path
+datadir=/home/yunohost.app/$app
+ynh_app_setting_set --app=$app --key=datadir --value=$datadir
```
Comment (alexAubin) [inline scripts/install:97]: Eeeh we probably want to define this new var in the upgrade script ?

**Comment by ericgaspar** [comment]
!testme

**Review by alexAubin — APPROVED** [review]
LGTM :+1:

**Comment by alexAubin** [comment]
(bumpitybump, shall we merge ? :eyes: )

**Pull request merged**